From 1e8a1f5b5ff8c5ba6601125151431178e68b71d4 Mon Sep 17 00:00:00 2001
From: Yuki <yuki.utk8g@gmail.com>
Date: Fri, 12 Jan 2024 09:31:54 +0300
Subject: [PATCH] Use bundled tgvoip

---
 Telegram/cmake/lib_tgvoip.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Telegram/cmake/lib_tgvoip.cmake b/Telegram/cmake/lib_tgvoip.cmake
index 886cd9935..9facda50e 100644
--- a/Telegram/cmake/lib_tgvoip.cmake
+++ b/Telegram/cmake/lib_tgvoip.cmake
@@ -7,15 +7,6 @@
 add_library(lib_tgvoip INTERFACE IMPORTED GLOBAL)
 add_library(tdesktop::lib_tgvoip ALIAS lib_tgvoip)
 
-if (DESKTOP_APP_USE_PACKAGED)
-    find_package(PkgConfig REQUIRED)
-    pkg_check_modules(TGVOIP IMPORTED_TARGET tgvoip)
-
-    if (TGVOIP_FOUND)
-        target_link_libraries(lib_tgvoip INTERFACE PkgConfig::TGVOIP)
-        return()
-    endif()
-endif()
 
 include(CMakeDependentOption)
 
-- 
2.43.0

