c573b65a Fix linking with LZMA when auto update is disabled
diff --git a/cmake/external/lzma/CMakeLists.txt b/cmake/external/lzma/CMakeLists.txt
new file mode 100644
index 00000000..8e75394d
--- a/cmake/external/lzma/CMakeLists.txt
+++ b/cmake/external/lzma/CMakeLists.txt
@@ -0,0 +1,19 @@
+# This file is inspired by Desktop App Toolkit,
+# a set of libraries for developing nice desktop applications.
+#
+# For license and copyright information please follow this link:
+# https://github.com/desktop-app/legal/blob/master/LEGAL
+
+if (DESKTOP_APP_USE_PACKAGED)
+    if (DESKTOP_APP_DISABLE_AUTOUPDATE)
+        add_library(external_lzma INTERFACE IMPORTED GLOBAL)
+        add_library(desktop-app::external_lzma ALIAS external_lzma)
+        find_package(PkgConfig REQUIRED)
+        pkg_check_modules(LZMA REQUIRED IMPORTED_TARGET liblzma)
+        target_link_libraries(external_lzma INTERFACE PkgConfig::LZMA)
+        return()
+    else()
+        # See auto_update directory contents
+    endif()
+    message(FATAL_ERROR "DEKSTOP_APP_USE_PACKAGED expected to be TRUE")
+endif()

